packages/api/src/hooks/useJurisdictionStatus.ts

```diff
@@ -16,90 +16,94 @@ type TServiceStatus = NonNullable<NonNullable<ReturnType<typeof usePOI>['data']>
-                    status.is_failed = true;
-                } else if (isStatusPending) {
-                    status.is_pending = true;
-                }
-        }
+            const status = {
+                is_failed: false,
+                is_not_applicable: false,
+                is_pending: false,
+                is_verified: false,
+            };
+            switch (jurisdiction) {
+                case 'bvi':
+                    if (
+                        isPOIServiceStatus('expired', 'rejected', 'suspected') ||
+                        authenticationStatus?.is_idv_revoked ||
+                        mt5_account_status === 'proof_failed'
+                    ) {
+                        status.is_failed = true;
+                    } else if (isPOIServiceStatus('pending') || isStatusPending) {
+                        status.is_pending = true;
+                    } else if (isPOIServiceStatus('verified')) {
+                        status.is_verified = true;
+                    }
+                    break;
+                case 'labuan':
+                    if (
+                        isPOIServiceStatus('expired', 'rejected', 'suspected') ||
+                        authenticationStatus?.is_idv_revoked ||
+                        // NOTE: BE plans to rename this to `authenticated_with_idv`
+                        authenticationStatus?.is_authenticated_with_idv_photoid ||
+                        mt5_account_status === 'proof_failed'
+                    ) {
+                        status.is_failed = true;
+                    } else if (isPOIServiceStatus('pending') || isStatusPending) {
+                        status.is_pending = true;
+                    } else if (isPOIServiceStatus('verified')) {
+                        status.is_verified = true;
+                    }
+                    break;
+                case 'svg':
+                    status.is_not_applicable = true;
+                    break;
+                default:
+                    // for other jurisdictions, MT5 account status already checks for POA status in BE
+                    if (mt5_account_status === 'proof_failed') {
+                        status.is_failed = true;
+                    } else if (isStatusPending) {
+                        status.is_pending = true;
+                    } else if (isPOIServiceStatus('verified')) {
+                        status.is_verified = true;
+                    }
+            }
 
-        return status;
-    }, [
-        jurisdiction,
-        poiStatus,
-        authenticationStatus?.is_idv_revoked,
-        authenticationStatus?.is_authenticated_with_idv_photoid,
-        mt5_account_status,
-    ]);
+            return status;
+        },
+        [poiStatus, authenticationStatus?.is_idv_revoked, authenticationStatus?.is_authenticated_with_idv_photoid]
+    );
 
     return {
-        data: verification_status,
+        /**
+         * Function to get verification status
```

Comment:
<3 documenting functions; typically its better to put such docs with function definition,

Reply:
jsdoc doesnt work when i put it in function definition :(